src/app/components/aboutPage/aboutPage.component.scss

```diff
@@ -0,0 +1,3 @@
+@import '../../styles/mixins/mixins.scss';
```

Comment:
no need of importing styles if they are not use within the file.

Reply:
applies to all occurences

Reply:
removed all unused style imports